use-timescale/services/i-o-boost.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
@esclymeTS ,  I have had a bit of a think about this. Would it be best to add this page to the https://docs.timescale.com/use-timescale/latest/data-tiering/ section and add links to it from the main page.